app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -0,0 +1,47 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    # Generic JWT authenticator that receive JWT vendor configuration and uses to validate that the authentication
+    # request is valid, and return conjur authn token accordingly
+    JwtAuthenticate ||= CommandClass.new(
+      dependencies: {
+        token_factory: TokenFactory.new
+      },
+      inputs: %i[jwt_configuration account username]
+    ) do
+      def call
```

Comment:
please add a new line before this method

Reply:
Added